docs/user-guide/data-reduction.md

```diff
@@ -0,0 +1,141 @@
+
+## Model
+
+The sample reflectivity is related to the intensity of neutron counts in the detector by the model
+
+$$
+I_{sam}(\lambda, j) = F(\theta(\lambda, j, \mu_{sam}), w_{sam}) \cdot R(Q(\lambda, \theta(\lambda, j, \mu_{sam}))) \cdot I_{ideal}(\lambda, j)
+$$ (model)
+where $I_{sam}(\lambda, j)$ represents the number of neutrons detected in the $j$ pixel of the detector having a wavelength in the interval $[\lambda, \lambda + d\lambda]$. $I_{ideal}$ represents the number of neutrons that would have been detected if the sample was a perfect reflector and large enough so that the footprint of the focused beam on the sample was small compared to the sample. $F(\theta, w)$ is the fraction of the focused beam that hits the sample. It depends on the incidence angle $\theta$ and on the size of the sample represented by $w$. $\mu_{sam}$ is the sample rotation.
+
+The ideal intensity is estimated from a reference measurement on a neutron supermirror.
+How it is computed will be described later, for now assume it exists.
+
+## Estimating $R(Q)$
+Move $F$ to the left-hand-side of equation {eq}`model` and integrate over all $\lambda$ and $j$ contributing to one particular $Q$-bin $[q_{i}, q_{i+1}]$
+
+$$
+\int_{Q(\lambda, \theta(\lambda, j, \mu_{sam})) \in [q_{i}, q_{i+1}]} \frac{I_{sam}(\lambda, j)}{F(\theta(\lambda, j, \mu_{sam}), w_{sam})} d\lambda \ dj = \\
+\int_{Q(\lambda, \theta(\lambda, j, \mu_{sam})) \in [q_{i}, q_{i+1}]}  I_{ideal}(\lambda, j) R(Q(\lambda, \theta(\lambda, j, \mu_{sam}))) d\lambda  \ dj.
+$$
+Notice that if the $Q$ binning is sufficiently fine then $R(Q)$ is approximately constant in the integration region.
+Assuming the binning is fine enough $R(Q)$ can be moved outside the integral and isolated so that
+
+$$
+ R(Q_{i+\frac{1}{2}}) \approx \frac{\int_{Q(\lambda, j, \mu_{sam}) \in [q_{i}, q_{i+1}]} \frac{I_{sam}(\lambda, j)}{F(\theta(\lambda, j, \mu_{sam}), w_{sam})} d\lambda \ dj }{\int_{Q(\lambda, \theta(\lambda, j, \mu_{sam})) \in [q_{i}, q_{i+1}]}  I_{ideal}(\lambda, j) d\lambda  \ dj} := \frac{I_{measured}(Q_{i+\frac{1}{2}})}{I_{ideal}(Q_{i+\frac{1}{2}})}
+$$
+for $Q_{i+\frac{1}{2}} \in [q_{i}, q_{i+1}]$.
+
+
+## The reference intensity $I_{ideal}$
+$I_{ideal}$ is estimated from a reference measurement on a neutron supermirror with known reflectivity curve.
+The reference measurement intensity is modeled the same way the sample measurement was
+
+$$
+I_{ref}(\lambda, j) = F(\theta(\lambda, j, \mu_{ref}), w_{ref}) \cdot R_{supermirror}(Q(\lambda, \theta(\lambda, j, \mu_{ref}))) \cdot I_{ideal}(\lambda, j)
+$$
+but in this case $R_{supermirror}(Q)$ is known.
+
+This leads to
+
+$$
+I_{ideal}(Q_{i+\frac{1}{2}}) = \int_{Q(\lambda, \theta(\lambda, j, \mu_{sam})) \in [q_{i}, q_{i+1}]} \frac{I_{ref}(\lambda, j)}{F(\theta(\lambda, j, \mu_{ref}), w_{ref}) R_{supermirror}(Q(\lambda, \theta(\lambda, j, \mu_{ref})))}
+ d\lambda  \ dj.
+$$
+
+## Estimating intensities from detector counts
+The neutron counts are Poisson distributed.
```

Comment:
This is true for counts in a 'spatial' binning. E.g., in pixel-tof-space. But is this also true after transforming to Q? It seems odd to me that we would get a Poisson distribution in Q bins. If this was true, we would not need to do any error propagation at all before we compute a Q histogram.

The reason is that binning in Q combines events from multiple pixel-tof bins in a non-trivial way.

Concerning your equations, $\sum_k$ *is* the expectation value. So you don't need an extra $E[...]$. Unless I misunderstand something?

Reply:
> It seems odd to me that we would get a Poisson distribution in Q bins. If this was true, we would not need to do any error propagation at all before we compute a Q histogram.

Yes my understanding is that the event counts should be Poisson distributed also in Q bins. The only assumptions that makes the count Poisson distributed are:
1. Constant (in time) expected event rate.
2. Events arrive independently from each other.

> If this was true, we would not need to do any error propagation at all before we compute a Q histogram.

I see what you mean. The events are weighted. So one event is actually not a single events, but a "package" of events. Individual packages of events arrive independently, but inside a package the events don't arrive independently.
I think we should talk about this in person to get the formulation right here. 👍


> Concerning your equations, is the expectation value. So you don't need an extra . Unless I misunderstand something?

The sum is an estimate of the expectation value. Not the expectation value itself.

Reply:
> The event count (the sum) is a random variable. We are interested in the expected value of that random variable because that's what's associated with the properties of the sample.

Yes, the count is the random variable. And the events are samples. So the sum is the sample mean, up to a normalisation factor. And thus, it is an estimator of the expectation value of the count. Basically,

$$
E[\sum_k x_k] = \sum_k E[x_k] = \sum_k w_k
$$

where $w_k$ s the weight of the events.